examples/models/llama2/export_llama_lib.py

```diff
@@ -46,6 +46,7 @@
 logging.basicConfig(level=logging.INFO, format=FORMAT)
 
 pkg_name = __name__
+verbosity_setting = None
```

Comment:
Might be good to make this clearly internal, though I don't know if anyone imports this file.

```suggestion
_verbosity_setting = None
```